src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -2,31 +2,44 @@
 import reaction_data as rxd
 import tendl_processing as tp
 import groupr_tools
+import file_handling as fh
+from pandas import concat
 
 def main():
     """
     Main method when run as a command line script.
     """
 
+    TAPE20 = 'tape20'
+    TAPE21 = 'tape21'
+
     mt_dict = rxd.process_mt_data(rxd.load_mt_table('mt_table.csv'))
+    cumulative_data = tp.cumulative_data
 
-    endf_path = 'tape20'
-    pendf_path = 'tape21'
+    for isotope, file_properties in fh.search_for_files().items():
+        element = file_properties['Element']
+        A = file_properties['Mass Number']
+        endf_path, pendf_path = file_properties['File Paths']
+        endf_path.rename(TAPE20)
+        pendf_path.rename(TAPE21)
 
-    material_id, MTs, endftk_file_obj = tp.extract_endf_specs(endf_path)
-    MTs = list(set(MTs).intersection(mt_dict.keys()))
-    njoy_input = groupr_tools.fill_input_template(material_id, MTs, 'Fe', 56, mt_dict)
-    groupr_tools.write_njoy_input_file(njoy_input)
-    gendf_path = groupr_tools.run_njoy('Fe', 56, material_id)
+        material_id, MTs, endftk_file_obj = tp.extract_endf_specs(TAPE20)
+        MTs = set(MTs).intersection(mt_dict.keys())
+        njoy_input = groupr_tools.fill_input_template(material_id, MTs,
+                                                      element, A, mt_dict)
+        groupr_tools.write_njoy_input_file(njoy_input)
+        gendf_path = groupr_tools.run_njoy(element, A, material_id)
 
-    pKZA = tp.extract_gendf_pkza(gendf_path)
-    # Extract MT values again from GENDF file as there may be some difference
-    # from the original MT values in the ENDF/PENDF files
-    material_id, MTs, endftk_file_obj = tp.extract_endf_specs(gendf_path)
-    gendf_data = tp.iterate_MTs(MTs, endftk_file_obj, mt_dict, pKZA)
-    gendf_data.to_csv('gendf_data.csv')
-    groupr_tools.cleanup_njoy_files()
+        pKZA = tp.extract_gendf_pkza(gendf_path)
+        # Extract MT values again from GENDF file as there may be some
+        # difference from the original MT values in the ENDF/PENDF files
+        material_id, MTs, endftk_file_obj = tp.extract_endf_specs(gendf_path)
+        gendf_data = tp.iterate_MTs(MTs, endftk_file_obj, mt_dict, pKZA)
+        cumulative_data = concat([cumulative_data, gendf_data],
+                                 ignore_index=True)
```

Comment:
Seeing how this is used across multiple nuclides, I'm not convinced that a pandas dataframe offers much more than a list of dictionaries.  I think `gendf_data` could look like:
```
[ {'Parent KZA' : pkza, 'Daughger KZA' : dkza, etc.... } , 
  {'Parent KZA' : pkza, 'Daughger KZA' : dkza, etc.... } ,  ... ]
```
and `cumulative_data` could be a local variable that gets appended with the new data.  It will require a change to `iterate_MTs()` that I think will make that a little simpler too.

At the end, this list could be used to create a new dataframe and export to CVS:
```
pd.DataFrame(cumulative_data).to_csv(filename)
```